src/i18n/en.json

```diff
@@ -2485,13 +2485,14 @@
     "end_date": "End date",
     "sign_reloaded": "Sign reloaded",
     "date_published": "Date published",
+    "event_jump": "Time has been jumped to the selected activity",
     "template_published": "Template published",
     "static_saved": "Static banner updated",
     "placeholders": {
       "locations": "Select location",
       "template": "Select template",
       "date": "Select date",
-      "search": "Search signs"
+      "search": "Search for activities"
```

Comment:
i do think that search place holder should depend on the context ( tab selected)
* Activities
* Banners
so u should be using a replacement variable here